README.md

```diff
@@ -30,7 +30,7 @@ $ npm install @mrshmllw/smores-react
 
 When making changes and creating PR's we use [semantic-releases](https://www.npmjs.com/package/semantic-release) which make use of [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/). 
 
-So to ensure our CHANGELOG.md is updated automatically and gets the changes we have made, we just need to name our PR's following this convention: e.g
+So to ensure our CHANGELOG.md is updated automatically and gets the changes we have made, we just need to name our PR's following this convention (casing matters): e.g
```

Comment:
Would this change benefit in being separated and wrapped in an alert?

Using something like this:
> [!WARNING]
> Commit message prefixes are case sensitive

Other alerts are available here:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
Above also works when rendering the md on vscode